src/MonitoringFunctions.Linux/DryRunUrlChecker.cs

```diff
@@ -7,7 +7,7 @@
 namespace MonitoringFunctions.Linux
 {
     /// <summary>
-    /// Runs the shell script in -DryRun mode and checks weather the generated links are accessible
+    /// Runs the shell script in -DryRun mode and checks whether the generated links are accessible
```

Comment:
nit: I believe that the typo exists in the Windows version of this file as well.